internal/testhelpers/tagsuite/tagsuite.go

```diff
@@ -8,6 +8,45 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+const (
+	Activate      = "activate"
+	Alternative   = "alternative"
+	Auth          = "auth"
+	CLIDeploy     = "cli-deploy"
+	Condition     = "condition"
+	Critical      = "critical"
+	Deploy        = "deploy"
+	Edit          = "edit"
+	Error         = "error"
+	Events        = "events"
+	Export        = "export"
+	Fork          = "fork"
+	History       = "history"
+	Init          = "init"
+	Interrupt     = "interrupt"
+	JSON          = "json"
+	Komodo        = "komodo"
+	Languages     = "languages"
+	MSI           = "msi"
+	Organizations = "organizations"
+	Output        = "output"
+	Package       = "package"
+	Perl          = "perl"
+	Platforms     = "platforms"
+	Prepare       = "prepare"
+	Pull          = "pull"
+	Push          = "push"
+	Python        = "python"
+	Run           = "run"
+	Scripts       = "scripts"
+	Secrets       = "secrets"
+	Shell         = "shell"
+	Shim          = "shim"
+	Show          = "show"
+	Update        = "Update"
```

Comment:
```suggestion
	Update        = "update"
```

All the other ones are lowercase.